src/services/proxies/bulk-resolver.service.ts

```diff
@@ -7,6 +7,15 @@ export class BulkProxy<C extends any, R extends ResolverService> implements Reso
 
   private readonly configuredResolvers: R[] = [];
 
+  private readonly notFoundMessages = [
+    "Cant resolve",
+    "Invalid characters, allowed only lowercase alphanumeric and -_",
+    "is not registered",
+    "is invalid",
+    "is not supported",
+    "Invalid name"
+  ]
```

Comment:
`Incorrect domain` from lens.
`No records found for domain` from redefined

Also need to support all errors from our contracts
https://github.com/e2xlabs/redefined-connect-decentralized/blob/master/email/contracts/EmailNameService.sol
https://github.com/e2xlabs/redefined-connect-decentralized/blob/master/nickname/contracts/NickNameService.sol

Reply:
I found only one mistake in the contract in function resolve(): "Name is not registered".
Other errors added.

Reply:
Nicknames
![image](https://user-images.githubusercontent.com/104828301/235604758-c5bc0f0a-8195-4259-964d-c45e79dd3ea5.png)
![image](https://user-images.githubusercontent.com/104828301/235604793-3f9ac2e3-d360-45e5-b036-f2463039163c.png)

Emails
![image](https://user-images.githubusercontent.com/104828301/235604888-ec180cac-8229-40bb-af35-46a4725ddc5b.png)
![image](https://user-images.githubusercontent.com/104828301/235604913-ff7e6023-d02d-4533-837f-ee324a41d026.png)

Reply:
sry, fixed

Reply:
Errors should be splitted by vendor, because in some theoretical cases error A can be "notFoundMessage" for one vendor and be OK for others. I think we can just use map like
Vendor.A : ["lol", "kek"] etc